src/QuadForm/LineOrbits.jl

```diff
@@ -780,3 +780,4 @@ __isless(::Type{T}) where {T} = _isless
 ################################################################################
 
 defining_polynomial(F::FqFiniteField) = minpoly(gen(F))
+
```

Comment:
Out of curiosity, why did you add newlines at the end of files?

Reply:
I guess I got told this in one of my PRs on Oscar or Hecke at my very beginning, and I thought it was of usual use in the guide-style. I changed them because I thought it was forgotten and should be added. :)

Reply:
I think there is a mixup between terms. Every file needs to end with a "newline" character, but this does not mean the it must end with a "new line". See for example https://thoughtbot.com/blog/no-newline-at-end-of-file

Reply:
Hum ok, I am sure whether I understand what is a "newline" character, but well, this is certainly coming from this confusion. Is it something wrong I should fix or is it fine like this?

Reply:
It is fine, you can leave it. 

P.S.: There are only a few files in Hecke missing newline characters at the end:
```
./test/Misc/FiniteField.jl
./test/NumFieldOrd.jl
./test/NfOrd/TorsionUnits.jl
./system/Build.jl
./src/Misc/Plesken.jl
./src/Misc/FactoredElemIntegers.jl
./src/NumFieldOrd/NfOrd/Unit/Helper.jl
./src/NumFieldOrd/NfOrd/FracIdeal.jl
./src/NumFieldOrd/NfRelOrd/FracIdeal.jl
./src/LocalField/Types.jl
./src/FieldFactory/non_normal.jl
./src/FieldFactory/rayclassgrp.jl
./src/NumField/promote_rules.jl
```